Pokemon/Pokemon/PokemonLibrary.m

```diff
@@ -0,0 +1,63 @@
+//
+//  PokemonLibrary.m
+//  Pokemon
+//
+//  Created by Brian Blanco on 7/31/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "PokemonLibrary.h"
+
+NSString *const kName = @"name";
+NSString *const kDescription = @"description";
+NSString *const kIcon = @"icon";
+NSString *const kLargeIcon = @"largeIcon";
+NSString *const kType = @"type";
+
+@implementation PokemonLibrary
+
+- (instancetype)init {
+    self = [super init];
+    if (self) {
+        //used to set up my PoKeMon dictionary
+        _library = @[@{kName: @"Pikachu",
```

Comment:
This is more of an aside, but in general, I like to define static data structures like this in a separate property list or JSON file, and then load them into memory as needed. It might be interesting for you to try your hand at loading this data from a file.